scripts/build-index.js

```diff
@@ -25,12 +25,27 @@ function buildPagesIndex(files) {
       if (!index[page].platform.includes(os)) {
         index[page].platform.push(os);
       }
-
       if (!index[page].language.includes(language)) {
         index[page].language.push(language);
       }
+      let targets = index[page].targets;
+      let needsPush = true;
+      for (const target of targets) {
+        if (target.platform === os && target.language === language) {
+          needsPush = false;
+          continue
+        }
+      }
+      if (needsPush) {
+        targets.push({"os": os, "language": language})
+        index[page].targets = targets;
+      }
     } else {
-      index[page] = {name: page, platform: [os], language: [language]};
+      index[page] = {name: page,
```

Comment:
```suggestion
      index[page] = {
        name: page,
```
Probably an oversight.